.github/workflows/simorgh-misc-checks.yml

```diff
@@ -29,6 +28,7 @@ jobs:
         run: npm ci
 
       - name: Chromatic UI Tests
+        if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
```

Comment:
This line ensures this step only runs if the pull request is opened up in the simorgh repo where it has access to the GitHub secrets